webpack/assets/javascripts/foreman_navigation.js

```diff
@@ -1,7 +1,19 @@
 import $ from 'jquery';
+import { contentSwaping } from './services/InternalAjax';
 import store from './react_app/redux';
 import * as LayoutActions from './react_app/components/Layout/LayoutActions';
 
+export const ApplyLinks = () => {
+  const anchorTags = document.querySelectorAll('a[data-ajax-loading');
```

Comment:
This looks incomplete
```suggestion
  const anchorTags = document.querySelectorAll('a[data-ajax-loading]');
```